network/src/application/netperf/interface.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+//
+use crate::protocols::network::{NetworkEvents, NetworkSender};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct NetPerfPayload {
+    byte: Vec<u8>,
+}
+
+impl NetPerfPayload {
+    pub fn new(mut len: usize) -> Self {
+        let mut v = Vec::with_capacity(len);
```

Comment:
nit: could this not just be:

```
let bytes = vec![0; len];
```
Would that work?

Also, I'd maybe rename `new` to something like `new_with_zeros` (to indicate we're just creating zero bytes).